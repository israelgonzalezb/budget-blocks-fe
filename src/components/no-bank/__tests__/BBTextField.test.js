
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import BBTextField from '../BBTextField';


describe('BBTextField test', () => {
  it('BBTextField should match snapshot', () => {
    const component = renderer.create(<BBTextField
       />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
