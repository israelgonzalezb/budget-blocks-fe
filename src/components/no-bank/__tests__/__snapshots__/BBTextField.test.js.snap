// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BBTextField test BBTextField should match snapshot 1`] = `
<div
  className="MuiFormControl-root MuiTextField-root"
>
  <div
    className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
    onClick={[Function]}
  >
    <input
      aria-invalid={false}
      autoFocus={false}
      className="MuiInputBase-input MuiOutlinedInput-input"
      disabled={false}
      onAnimationStart={[Function]}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      style={
        Object {
          "height": "5px",
        }
      }
      type="text"
    />
    <fieldset
      aria-hidden={true}
      className="PrivateNotchedOutline-root-39 MuiOutlinedInput-notchedOutline"
      style={
        Object {
          "paddingLeft": 8,
        }
      }
    >
      <legend
        className="PrivateNotchedOutline-legend-40"
        style={
          Object {
            "width": 0.01,
          }
        }
      >
        <span
          dangerouslySetInnerHTML={
            Object {
              "__html": "&#8203;",
            }
          }
        />
      </legend>
    </fieldset>
  </div>
</div>
`;
